src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/instances/common/inverse.cpp

```diff
@@ -16,7 +16,8 @@ const std::vector<std::vector<ov::test::InputShape>> input_shapes = {
 
 const auto shapes = testing::ValuesIn(input_shapes);
 
-const auto dtypes = testing::Values(ov::element::f32, ov::element::f16, ov::element::bf16);
+// todo: ov::element::f16 is temporary disabled
+const auto dtypes = testing::Values(ov::element::f32, ov::element::bf16);
```

Comment:
I think as a W.A. of disable these test case, better to create related tickets then skip it instead of delete it in order to keep the test case coverage.